shogun-lib/src/main/java/de/terrestris/shoguncore/controller/UserController.java

```diff
@@ -167,4 +181,44 @@ public void changePassword(@RequestBody PasswordChange passwordChangeBody) {
             );
         }
     }
+
+    @PostMapping(value = "/resetPassword")
+    public void resetPassword(@RequestBody PasswordReset passwordResetBody, HttpServletRequest request){
+        User user = service.getUserByEmail(passwordResetBody.getEmail());
+
+        // publish event that sends the email with link to confirm password change
+        // Publish event which sends a confirmation email to enable the user
+        try {
+            URI appUrl = HttpUtil.getApplicationURIFromRequest(request);
+            OnPasswordResetRequestEvent event = new OnPasswordResetRequestEvent(user, request.getLocale(), appUrl.toString());
+            eventPublisher.publishEvent(event);
+        } catch (MailException | URISyntaxException e) {
+            LOG.error("Error sending mail: {}", e.getMessage());
+            LOG.trace("Full stack trace: ", e);
+
+            throw new ResponseStatusException(
+                HttpStatus.BAD_REQUEST,
+                messageSource.getMessage("registration.error.mail", null, request.getLocale())
```

Comment:
Please set a custom message for this use case (e.g. `passwordReset.error.mail`).